src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -136,11 +136,19 @@ <h5>{% trans "Logic rules problems" %}</h5>
         <ul>
         {% for logic_rule in invalid_logic_rules %}
             <li>
+                {% if logic_rule.exception %}
+                    <p>
+                        {% blocktranslate with form_name=logic_rule.form_name trimmed %}
+                            We couldn't process the logic rule for '{{ form_name }}' because it is completely broken. <br>
```

Comment:
```suggestion
                            We couldn't process logic rule {{ index }} for '{{ form_name }}' because it appears to be invalid. <br>
```
and we'd also have to keep track of the index/order of the logic rule to display it. I think this is the best we can do to point them to the problematic rule.

I've also softened the language a bit :P